File: uploads/2025-08-08_18-52-05-182792/test5.java
package test_files.test5;
import java.util.List;

public class test5 {
    public static Result processNumbers(List<Integer> numbers) {
        int sumEven = 0;
        int maxNumber = numbers.get(0);

        for (int num : numbers) {
            if (num % 2 == 0) {
                sumEven += num;
            }
            if (num > maxNumber) {
                maxNumber = num;
            }
        }

        return new Result(sumEven, maxNumber);
    }
}

class Result {
    public int sumEven;
    public int maxNumber;

    public Result(int sumEven, int maxNumber) {
        this.sumEven = sumEven;
        this.maxNumber = maxNumber;
    }
}

File: outputs/2025-08-08_18-52-05-182792/test5.py
# Unit test result: ✅ PASSED
class test5:

    @staticmethod
    def process_numbers(numbers: list[int]) -> 'Result':
        sum_even: int = 0
        max_number: int = numbers[0]
        for num in numbers:
            if num % 2 == 0:
                sum_even += num
            if num > max_number:
                max_number = num
        return Result(sum_even, max_number)

class Result:
    sumEven: int
    maxNumber: int

    def __init__(self, sumEven: int, maxNumber: int):
        self.sumEven = sumEven
        self.maxNumber = maxNumber